src/app/functions/server/dashboard.test.ts

```diff
@@ -650,38 +650,39 @@ describe("getDashboardSummary", () => {
     );
     noNegativeCounts(summary);
     expect(summary.dataBreachTotalNum).toBe(0);
-    expect(summary.dataBreachTotalExposuresNum).toBe(0);
-    expect(summary.dataBreachFixedExposuresNum).toBe(0);
+    expect(summary.dataBreachTotalDataPointsNum).toBe(0);
+    expect(summary.dataBreachFixedDataPointsNum).toBe(0);
     expect(summary.dataBrokerTotalNum).toBe(4);
-    expect(summary.inProgressFixedExposures.emailAddresses).toBe(
-      summary.inProgressExposures.emailAddresses +
-        summary.fixedExposures.emailAddresses,
+    expect(summary.inProgressFixedDataPoints.emailAddresses).toBe(
+      summary.inProgressDataPoints.emailAddresses +
+        summary.fixedDataPoints.emailAddresses,
     );
-    expect(summary.inProgressFixedExposures.phoneNumbers).toBe(
-      summary.inProgressExposures.phoneNumbers +
-        summary.fixedExposures.phoneNumbers,
+    expect(summary.inProgressFixedDataPoints.phoneNumbers).toBe(
+      summary.inProgressDataPoints.phoneNumbers +
+        summary.fixedDataPoints.phoneNumbers,
     );
-    expect(summary.inProgressFixedExposures.addresses).toBe(
-      summary.inProgressExposures.addresses + summary.fixedExposures.addresses,
+    expect(summary.inProgressFixedDataPoints.addresses).toBe(
+      summary.inProgressDataPoints.addresses +
+        summary.fixedDataPoints.addresses,
     );
-    expect(summary.inProgressFixedExposures.familyMembers).toBe(
-      summary.inProgressExposures.familyMembers +
-        summary.fixedExposures.familyMembers,
+    expect(summary.inProgressFixedDataPoints.familyMembers).toBe(
+      summary.inProgressDataPoints.familyMembers +
+        summary.fixedDataPoints.familyMembers,
     );
-    expect(summary.unresolvedExposures.emailAddresses).toBe(0);
+    expect(summary.unresolvedDataPoints.emailAddresses).toBe(0);
   });
 
   it("gets scanned results manually removed summary", () => {
     const summary = getDashboardSummary(manuallyResolvedScannedResults, []);
     noNegativeCounts(summary);
     expect(summary.dataBreachTotalNum).toBe(0);
-    expect(summary.dataBreachTotalExposuresNum).toBe(0);
-    expect(summary.dataBreachFixedExposuresNum).toBe(0);
+    expect(summary.dataBreachTotalDataPointsNum).toBe(0);
+    expect(summary.dataBreachFixedDataPointsNum).toBe(0);
     expect(summary.dataBrokerTotalNum).toBe(1);
-    expect(summary.dataBrokerFixedNum).toBe(0);
-    expect(summary.totalExposures).toBe(summary.dataBrokerTotalExposuresNum);
-    expect(summary.dataBrokerManuallyResolvedExposuresNum).toBe(12);
-    expect(summary.unresolvedExposures.emailAddresses).toBe(3);
+    expect(summary.dataBrokerAutoFixedNum).toBe(0);
+    expect(summary.totalDataPoints).toBe(summary.dataBrokerTotalDataPointsNum);
+    expect(summary.dataBrokerManuallyResolvedDataPointsNum).toBe(12);
+    expect(summary.unresolvedDataPoints.emailAddresses).toBe(0);
```

Comment:
Q: Why did this change from 3 to 0?

```diff
-expect(summary.unresolvedExposures.emailAddresses).toBe(3);
+expect(summary.unresolvedDataPoints.emailAddresses).toBe(0);
```

Reply:
Heh, that is the

> Sort of - I fixed an existing incorrect unit test.

That test at some point was incorrectly [updated to check for three email addresses](https://github.com/mozilla/blurts-server/commit/72e443149559dd5aca40d81c2c014e0ef12d72c7#diff-08247bd80738f0809f780458b256238ac468b0ce54ed53895f42a5572f21b6bfR684), presumably to match the test output to quickly resolve a failing test - but since all the scan results in that tests have been manually resolved, there should be 0 unresolved email addresses left.

But excellent callout! If we had these eagles eyes in the initial PR, we wouldn't have had this bug.